Assets/Scripts/Scenes/Lobby/RoomSettingPanel/LookDisplayObject.cs

```diff
@@ -0,0 +1,128 @@
+        [SerializeField] private GameObject�@_uiPanel;
+        [SerializeField] private NetworkRunner _runnerPrefab;
+        [SerializeField] private float _motionTime = 1f;
+
+        private int _isOpened = 0; //�f�B�X�v���CUI�̕\�����
+        private Vector3 _displayPosition = Vector3.zero; //�f�B�X�v���C��Transform
+        private Quaternion _displayRotation = Quaternion.identity;
+
+        private void Start()
+        {
+            _displayPosition = _displayCamera.transform.position;
+            _displayRotation = _displayCamera.transform.rotation;
+        }
+
+        private void Update()
+        {
+            if (Input.GetKeyDown(KeyCode.E))
+            {
+                OnEnableDisplay();
+            }
+        }
+
+        /// <summary>
+        /// �f�B�X�v���C�ɒ��ڂ���
+        /// </summary>
+        public async void OnEnableDisplay()
+        {
+            //������񂵂��ʂ�Ȃ��悤�ɐ���
+            if (_isOpened != 0) return;
+            else _isOpened = 1;
+
+            //���[�V�����O����
+            _displayCamera.enabled = true;
+
+            //�����ҋ@
+            await UniTask.WhenAll(
+                CameraMotion(Camera.main.transform.position, //�J�������[�V����
+                Camera.main.transform.rotation,
+                _displayPosition,
+                _displayRotation),
+                BootRunner()); //NetworkRunner�N��
+
+            //���[�V�����㏈��
+            Cursor.visible = true;
+            Cursor.lockState = CursorLockMode.None;
+            _uiPanel.SetActive(true);
+            _isOpened = 2;
+        }
+
+        /// <summary>
+        /// ���_��߂�
+        /// </summary>
+        public async void OnDisableDisplay()
+        {
+            //������񂵂��ʂ�Ȃ��悤�ɐ���
+            if (_isOpened != 2) return;
+            else _isOpened = 1;
+
+            //���[�V�����O����
+            Cursor.visible = false;
+            Cursor.lockState = CursorLockMode.Locked;
+            _uiPanel.SetActive(false);
+            DiscardRunner(); //Runner�̒�~
+
+            //�����ҋ@
+            await CameraMotion(_displayPosition, _displayRotation, Camera.main.transform.position, Camera.main.transform.rotation);
+
+            //���[�V�����㏈��
+            _displayCamera.enabled = false;
+            _isOpened = 0;
+        }
+
+        private async UniTask CameraMotion(Vector3 startPosition, Quaternion startRotation, Vector3 endPosition, Quaternion endRotation)
+        {
+            float timer = 0f;
+            while (timer < _motionTime)
+            {
+                timer += Time.deltaTime; //�^�C�}�[�v��
+                _displayCamera.transform.position = Vector3.Slerp(startPosition, endPosition, timer / _motionTime); //���[�V����
+                _displayCamera.transform.rotation = Quaternion.Slerp(startRotation, endRotation, timer / _motionTime);
+                await UniTask.Yield(PlayerLoopTiming.Update); //1�t���[���҂�
+            }
```

Comment:
```suggestion
            _displayCamera.transform.DOMove(endPosition, _motionTime).From(startPosition).SetEase(Ease.InOutSine);
            _displayCamera.transform.DORotate(startRotation.eulerAngles, _motionTime).From(endRotation.eulerAngles).SetEase(Ease.InOutSine);
```
DoTweenで制御した方がいろいろ弄れて簡単だと思います。今回だと、SetEaseを調整したらカメラの遷移の仕方を帰れたりするので。